### PR TITLE
Include `projectName` in audit logs

### LINKED ISSUE #9875: Include project name in Database audit logs
Follow up of #9873 

We want to include project name, e.g. `enso_dev.Base_Tests` in the metadata of audit logs for Database connections.

Currently this is blocked by #9845.

Once the above ticket is resolved, we should update the logic and tests.

### PR DESCRIPTION
**Pull request by radeusgd**
### Pull Request Description

- Closes #9875 

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [x] Unit tests have been written where possible.

### CODE REVIEW

**Comment by radeusgd** [comment]
In 69fea889515527828193b80ced3f73fcf76272ea I also renamed metadata properties to be consistent with each other - all are now in `camelCase`. I thought it's a good time to do such a rename, before we start depending on them.

@jdunkerley does that seem OK?

**Review by GregoryTravis — APPROVED** [review]

File: std-bits/base/src/main/java/org/enso/base/CurrentEnsoProject.java [hunk 1]
```diff
@@ -1,19 +1,41 @@
 package org.enso.base;
 
+import org.enso.base.polyglot.EnsoMeta;
+import org.graalvm.polyglot.Value;
+
 /** A Java interface to the `Enso_Project` type. */
 public final class CurrentEnsoProject {
   private final String name;
   private final String namespace;
```
Comment (jdunkerley) [inline std-bits/base/src/main/java/org/enso/base/CurrentEnsoProject.java:9]: Use a record class?

**Review by jdunkerley — APPROVED** [review]

**Comment by jdunkerley** [comment]
> In [69fea88](https://github.com/enso-org/enso/commit/69fea889515527828193b80ced3f73fcf76272ea) I also renamed metadata properties to be consistent with each other - all are now in `camelCase`. I thought it's a good time to do such a rename, before we start depending on them.
> 
> @jdunkerley does that seem OK?

seems good to me.

File: std-bits/base/src/main/java/org/enso/base/CurrentEnsoProject.java (hunk 1 above)
Comment (radeusgd) [reply std-bits/base/src/main/java/org/enso/base/CurrentEnsoProject.java:9]: `record` implies public constructor, but I wanted to keep the constructor private, so the only way to get an instance of this class is through `CurrentEnsoProject.get()`.

**Pull request merged**